datasets/tess.yaml

```diff
@@ -0,0 +1,24 @@
+Name: Transiting Exoplanet Survey Satellite (TESS)
+Description: |
+  The Transiting Exoplanet Survey Satellite (TESS) is a two-year survey that will discover exoplanets in orbit around bright stars.
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Monthly
+Tags:
+  - astronomy
+  - aws-pds
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files in a 'Requester Pays' bucket
```

Comment:
Can you add a link to the RP docs here? The field accepts markdown so you can just do [Requester Pays](https://docs.aws.amazon.com/AmazonS3/latest/dev/RequesterPaysBuckets.html)